toolkit/javascripts/option-select.js

```diff
@@ -80,10 +80,13 @@
   };
 
   OptionSelect.prototype.checkedString = function checkedString(){
-    var count = this.$options.filter(":checked").size();
     var checkedString = "";
-    if (count > 0){
-      checkedString = count+" selected";
+
+    if (this.isClosed()) {
```

Comment:
No, it doesn't. If it's closed, the added line (102) in toggleOptionSelect triggers the update when it's actually closed.